features/dapp_develop/nodejs.go

```diff
@@ -0,0 +1,48 @@
+package dapp_develop
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/go-cmd/cmd"
+
+	e2e "github.com/stellar/system-test"
+)
+
+// return the fn response as a serialized string
+// uses secret-key and network-passphrase directly on command
+func invokeContractFromNodeJSTool(deployedContractId, contractName, functionName, param1 string, e2eConfig *e2e.E2EConfig) (string, error) {
+	args := []string{
+		"run",
+		"-s",
+		"invoke",
+		"--",
+		"--id", deployedContractId,
+		"--rpc-url", e2eConfig.TargetNetworkRPCURL,
+		"--account", e2eConfig.TargetNetworkPublicKey,
+		"--secret-key", e2eConfig.TargetNetworkSecretKey,
+		"--network-passphrase", e2eConfig.TargetNetworkPassPhrase,
+		"--fn", functionName,
```

Comment:
```suggestion
		"--", 
		functionName,
```

Reply:
That's not how the `invoke.ts` script works.